docs/gettingstarted/quickstart.rst

```diff
@@ -0,0 +1,268 @@
+      | Links, ports and volumes work across hosts                               |
+      +--------------------------------------------------------------------------+
+      | After deployment, containers can move around                             |
+      +--------------------------------------------------------------------------+
+
+      .. empty-div:: arrow-down center-block
+
+   .. parallel::
+
+      .. mobile-label::
+
+         Local
+
+
+      Installation
+      ------------
+
+      .. noscript-content::
+      
+         OS X
+         ^^^^
+
+         Install the flocker-cli client on your Mac (requires Homebrew):
+
+         .. code-block:: console
+
+            $ brew update && \
+              brew tap clusterhq/flocker && \
+              brew install flocker-0.3.2
+            
+      .. noscript-content::
+
+         Linux
+         ^^^^^
+
+         Install the flocker-cli client on your Linux machine:
+
+         .. code-block:: console
+
+            $ virtualenv flocker-tutorial && \
+              flocker-tutorial/bin/pip install --upgrade pip && \
+              flocker-tutorial/bin/pip install --quiet https://archive.clusterhq.com/downloads/flocker/Flocker-0.3.2-py2-none-any.whl
+
+      .. tabs::
+
+         OS X
+         ^^^^
+
+         Install the flocker-cli client on your Mac (requires Homebrew):
+
+         .. code-block:: console
+
+            $ brew update && \
+              brew tap clusterhq/flocker && \
+              brew install flocker-0.3.2
+
+         Linux
+         ^^^^^
+
+         Install the flocker-cli client on your Linux machine:
+
+         .. code-block:: console
+
+            $ virtualenv flocker-tutorial && \
+              flocker-tutorial/bin/pip install --upgrade pip && \
+              flocker-tutorial/bin/pip install --quiet https://archive.clusterhq.com/downloads/flocker/Flocker-0.3.2-py2-none-any.whl
+
+      .. empty-div:: arrow-down center-block invisible
+
+   .. parallel::
+
+      .. mobile-label::
+
+         Live
+
+
+      Installation
+      ------------
+
+      .. noscript-content::
+
+         Vagrant
+         ^^^^^^^
+
+         Simulate a Flocker cluster with virtual machines on your laptop (requires Vagrant, VirtualBox):
+
+         .. code-block:: console
+
+            $ git clone \
+              https://github.com/clusterhq/vagrant-flocker && \
+              cd vagrant-flocker && \
+              vagrant up
+
+         AWS
+         ^^^
+
+         Please see our separate `AWS install instructions <http://docs.clusterhq.com/en/latest/gettingstarted/installation.html#using-amazon-web-services>`_ to get started.
+
+      .. tabs::
+
+         Vagrant
+         ^^^^^^^
+
+         Simulate a Flocker cluster with virtual machines on your laptop (requires Vagrant, VirtualBox):
+
+         .. code-block:: console
+
+            $ git clone \
+              https://github.com/clusterhq/vagrant-flocker && \
```

Comment:
I fixed this by running this blob from our other Fedora install docs:

```
UNAME_R=$(uname -r)
PV=${UNAME_R%.*}
KV=${PV%%-*}
SV=${PV##*-}
ARCH=$(uname -m)
yum install -y https://kojipkgs.fedoraproject.org/packages/kernel/${KV}/${SV}/${ARCH}/kernel-devel-${UNAME_R}.rpm
```

With the right kernel headers package installed the VirtualBox install succeeded for real instead of only pretending to succeed.

I don't know if all of these instructions fit (or if we even want to recommend people install kernel headers from koji).  Maybe instead we should just make AWS the default tab and hope that only people who already like Vagrant and have it working click on the Vagrant tab.